lib/matplotlib/transforms.py

```diff
@@ -2436,8 +2463,8 @@ def __init__(self, boxin, boxout, **kwargs):
         Create a new :class:`BboxTransform` that linearly transforms
         points from *boxin* to *boxout*.
         """
-        assert boxin.is_bbox
-        assert boxout.is_bbox
+        if not boxin.is_bbox or not boxout.is_bbox:
+            msg = "'boxin' and 'boxout' must be bbox"
```

Comment:
and not quite what this is testing (this is just testing that they are 'true-ish' not that they are `bbox` instances).